### PR TITLE
Replacing Fonts with Avenir CDN

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Replacing the off-repository hosted fonts with CDN hosted fonts that will work on consumerfinance.gov and internal sites. Font css is based off [cf-core](https://github.com/cfpb/cf-core/blob/gh-pages/src/licensed-fonts.css).

## Changes
- Changes to the fonts.less and related mixins.less files.

## Review
- @ascott1 
- @Scotchester 

## Notes
Browser Testing

**IE 11**
image

**IE 10**
image

**IE 9**
image

### CODE REVIEW

File: regulations/static/regulations/css/less/mixins.less [hunk 1]
```diff
@@ -33,27 +33,27 @@ mixins.less contains any mixins or variables to be used throughout the LESS file
...
-    font-family: "Avenir Next Demi", Arial, sans-serif;
+    font-family: "AvenirNextLTW01-Demi", Arial, sans-serif;
     font-weight: normal;
     font-style: normal;
 }
 
 .avenir-next-demi-italic {
-    font-family: "Avenir Next Demi Italic", Arial, sans-serif;
+    font-family: "AvenirNextLTW01-DemiItalic", Arial, sans-serif;
     font-weight: normal;
```
Comment (Scotchester) [inline regulations/static/regulations/css/less/mixins.less]: Here and on L51, `font-weight` needs to be set to `bold` or `700`. See example in the mixins at the top of [cf-base.less](https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-base.less).

**Comment by Scotchester** [comment]
This is kinda minor, but could you move the `@font-face` blocks for Avenir Next Medium between the Regular and Demi blocks, so they go in order of weight?

File: regulations/static/regulations/css/less/mixins.less (hunk 1 above)
Comment (KimberlyMunoz) [inline regulations/static/regulations/css/less/mixins.less]: Good catch! I went what was off our font file before, so it's a good thing we're updating.

File: regulations/templates/regulations/base.html [hunk 2]
```diff
@@ -41,9 +41,6 @@
         <!--[if lt IE 9]>
             <script src="{%static "regulations/js/built/lib/respond/respond.min.js" %}"></script>
         <![endif]-->
-        <!--[if IE 7]>
-            <link rel="stylesheet" href="{%static "regulations/css/icon-ie7.css" %}"/>
```
Comment (KimberlyMunoz) [inline regulations/templates/regulations/base.html]: I forgot to remove this in #697.

**Comment by KimberlyMunoz** [comment]
Made the fixes. Thanks for the extra set of :eyes:, @Scotchester.

**Comment by ascott1** [comment]
:+1:

**Pull request merged**